pkgs/development/python-modules/harlequin-mysql/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  poetry-core,
+  python3Packages,
+}:
+
+buildPythonPackage rec {
+  pname = "harlequin-mysql";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "harlequin_mysql";
+    inherit version;
+    hash = "sha256-Ru9CxbZYVo9TQO5TwkHLEzPz4EkUgHwfg3Qeg1F4eLM=";
+  };
+
+  build-system = [
+    poetry-core
+  ];
+
+  dependencies = with python3Packages; [
```

```suggestion
  dependencies = [
```
Please use `mysql-connector` directly.
`python3Packages` is currently an alias for `python312Packages`, so it will break other version package sets, for example `python311Packages.harlequin-mysql`.

---

Seems there is no direct mysql-connector package, shall i change to ` dependencies = with python312Packages; [`?

---

Did you add it to the inputs?